libs/pages/clusters/src/lib/feature/page-new-feature/page-new-feature.tsx

```diff
@@ -0,0 +1,407 @@
+
+    return (
+      <button
+        className="flex gap-6 rounded border border-neutral-200 p-5 shadow-sm transition hover:bg-neutral-100"
+        onClick={() => {
+          posthog.capture('select-cluster', {
+            selectedCloudProvider,
+            selectedInstallationType,
+          })
+          openInstallationGuideModal()
+        }}
+      >
+        <div className="flex items-center">
+          {typeof icon === 'string' ? (
+            <img className="max-h-10 w-14 select-none" src={icon} alt={title} />
+          ) : (
+            cloneElement(icon as ReactElement, { className: 'w-10' })
+          )}
+        </div>
+        <div className="w-60 text-start">
+          <h3 className="mb-1 text-ssm font-medium">{title}</h3>
+          {selectedInstallationType === 'demo' && (
+            <span className="relative -top-0.5 inline-block rounded bg-brand-500 px-1 text-2xs text-neutral-50">
+              3min to setup
+            </span>
+          )}
+        </div>
+      </button>
+    )
+  }
+}
+
+export function PageNewFeature() {
+  const { organizationId = '' } = useParams()
+  useDocumentTitle('Create new cluster - Qovery')
+  const { openModal, closeModal } = useModal()
+
+  const openInstallationGuideModal = ({ isDemo = false }: { isDemo?: boolean } = {}) =>
+    openModal({
+      options: {
+        width: 500,
+      },
+      content: (
+        <ClusterInstallationGuideModal
+          mode="CREATE"
+          isDemo={isDemo}
+          type="ON_PREMISE"
+          onClose={() => {
+            closeModal()
+          }}
+        />
+      ),
+    })
+
+  return (
+    <Section className="flex w-full flex-1 flex-col gap-8 rounded-t bg-white p-8 pb-24">
+      <Link color="brand" to={CLUSTERS_URL(organizationId)} className="text-sm">
+        <Icon iconName="arrow-left" className="mr-1" />
+        Back to clusters
+      </Link>
+      <div className="mx-auto flex w-[1024px] flex-col gap-10">
+        <Heading className="mb-2 text-center text-2xl">Install cluster</Heading>
+        <Section className="grid grid-cols-3 gap-4">
+          <div className="col-span-3 gap-2">
+            <Heading>Qovery on your local machine</Heading>
+            <p className="text-xs text-neutral-350">Quickly test and validate the Qovery solution on your computer.</p>
+          </div>
+          <CardCluster
+            title="Local machine (demo)"
+            icon={Kubernetes}
+            selectedCloudProvider="OTHER"
+            selectedInstallationType="demo"
+            openInstallationGuideModal={() => openInstallationGuideModal({ isDemo: true })}
+          />
+        </Section>
+        <Section className="grid grid-cols-3 gap-4">
+          <div className="col-span-3 gap-2">
+            <Heading>Or choose your hosting mode</Heading>
+            <p className="text-xs text-neutral-350">Manage your infrastructure across different hosting mode.</p>
+          </div>
+          <CardCluster
```

Comment:
Could you use an array of object to generate list of cloud providers ?

Reply:
https://github.com/Qovery/console/pull/1697/commits/499f706ba53a6daded7adecb21863fe769531dbc